lms/static/sass/_build-discussion-editor.scss

```diff
@@ -0,0 +1,4 @@
+// From _build-course.scss
+
+@import "course/discussion/form-wmd-toolbar";
```

Comment:
Can this just be moved into _build-discussion.scss? It seems to always be used together.